thinkhazard/templates/report.jinja2

```diff
@@ -108,34 +108,34 @@
             {% endif %}
           </div>
           {% if hazard %}
-          <div class="text-center">
+          <div id="level" class="text-center">
```

Comment:
I'd prefer we don't use `id` here.
And the name should be more specific. "level" is too generic.

I'd rather use:
```html
class="caption level"
```
With the following in the less file:
```css
#map-block .caption.level {
}
```